templates/shared/contextual_footers/_download_all_installation.html

```diff
@@ -4,6 +4,6 @@ <h3 class="p-heading--four">Installation guides</h3>
   <ul class="p-list">
       <li><a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'install desktop guide', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installing Ubuntu Desktop</a></li>
       <li><a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-server" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'install server guide', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installing Ubuntu Server</a></li>
-      <li><a href="/download/cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'install autopilot', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installing OpenStack Autopilot&nbsp;&rsaquo;</a></li>
+      <li><a href="/download/cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'install openstack', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Install OpenStack&nbsp;&rsaquo;</a></li>
```

Comment:
Shouldn't this link to the OpenStack page? https://www.ubuntu.com/openstack

Reply:
but it's about installing?